policyengine_us/variables/gov/states/or/tax/income/credits/wfhdc/or_working_family_household_and_dependent_care_credit.py

```diff
@@ -26,7 +26,7 @@ def formula(tax_unit, period, parameters):
         match_percentage = p.match[income_category][eligibility_category]
 
         # Get the relevant expenses.
-        expenses = tax_unit("cdcc_relevant_expenses", period)
```

Comment:
Some confusions here @PavelMakarchuk 

Was wondering if the `cdcc_relevant_expenses` includes the expense for the disabled spouse? Based on the structure it only considered the childcare expenses. However, this WFHDC expense should include all expenses for qualified individuals

Also, do we need to consider the filer's employment status? Only single students or married working heads of household are eligible

![image](https://github.com/PolicyEngine/policyengine-us/assets/87620347/d4e94fd1-0eb4-43d0-b5d4-dbd649a98228)

Reply:
yes, I should move away from `cdcc_relevant_expenses` as it does not sufficiently covers the needed expenses 

For the employment income check we should look for earned_income > 0 , that makes sense given the name of the credit. We should also include the student / disability check, thank you for pointing that out